client/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "superdesk-client",
   "license": "GPL-3.0",
   "dependencies": {
-    "superdesk-core": "superdesk/superdesk-client-core#b3988c8c0b9aeb08896d03ca489588529a41aa78",
+    "superdesk-core": "vladnicoara/superdesk-client-core#48346a37115f2e38e4325ed76785136cdbccd862",
```

Comment:
For all PR in client core we need to validate against e2e tests.
Petr will move the running of e2e tests on client core so this is a temporary solution. 
I set with wip label all those PR.
@naomiaro If you think that a improvement is still needed you can try to do it as an example with your PR:
https://github.com/superdesk/superdesk-client-core/pull/136